test/pytest/test_code.py

```diff
@@ -27,4 +27,5 @@
 @pytest.mark.skipif(not shutil.which('mypy'), reason='mypy is not installed')
 def test_bridge_mypy():
     # only test src/cockpit; src/systemd_ctypes does not have type annotations yet
+    # disable caching, as it otherwise often crashes with "Cannot find component 'inotify' for 'systemd_ctypes...."
```

Comment:
This is just a comment, where is the actual change?

Reply:
git pilot error.. Pushed the actual fix :see_no_evil:

Reply:
Thanks!